CalibTracker/SiStripESProducers/plugins/fake/SiStripThresholdFakeESSource.cc

```diff
@@ -66,7 +66,7 @@ SiStripThresholdFakeESSource::produce(const SiStripThresholdRcd& iRecord)
 {
   using namespace edm::es;
 
-  std::shared_ptr<SiStripThreshold> threshold{new SiStripThreshold};
+  std::shared_ptr<SiStripThreshold> threshold = std::make_shared<SiStripThreshold>();
```

Comment:
The title of this PR is to move to unique_ptr. Can you quickly explain, why this is not extended to all conditions?